dev/start-etcd.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+export STORAGE_CONTAINER=test-copy-operation-plamen
+export GOOGLE_APPLICATION_CREDENTIALS=dev/gcp/serviceaccount.json
+
+export DEST_STORAGE_CONTAINER=test-copy-operation-dest-plamen
+export DEST_GOOGLE_APPLICATION_CREDENTIALS=dev/gcp-dest/serviceaccount.json
```

Comment:
Let's have SOURCE_ instead of DEST_ variables, it will be simpler to adapt the chart in `etcd-druid` this way.